erts/doc/src/erlang.xml

```diff
@@ -1654,33 +1764,82 @@ b</pre>
+            <c>P</c> exits with exit reason <c><anno>Reason</anno></c>.</p>
           </item>
-          <item><p>If <c><anno>Pid</anno></c> is trapping exits, the exit
-            signal is transformed into a message
-            <c>{'EXIT', From, <anno>Reason</anno>}</c>
-            and delivered to the message queue of <c><anno>Pid</anno></c>.</p>
-          </item>
-          <item><p><c>From</c> is the process identifier of the process
-            that sent the exit signal. See also
-            <seemfa marker="#process_flag/2">
-            <c>process_flag/2</c></seemfa>.</p>
+          <item><p>If <c>P</c> is <seemfa
+          marker="#process_flag/2">trapping exits</seemfa>, the exit
+          signal is transformed into a message <c>{'EXIT', From,
+          <anno>Reason</anno>}</c>, where <c>From</c> is the process
+          identifier of the process that sent the exit signal, and
+          delivered to the message queue of <c>P</c>.</p>
           </item>
         </list>
-        <p>If <c><anno>Reason</anno></c> is the atom <c>normal</c>,
-          <c><anno>Pid</anno></c>
-          does not exit. If it is trapping exits, the exit signal is
-          transformed into a message <c>{'EXIT', From, normal}</c>
-          and delivered to its message queue.</p>
-        <p>If <c><anno>Reason</anno></c> is the atom <c>kill</c>,
-          that is, if <c>exit(<anno>Pid</anno>, kill)</c> is called,
-          an untrappable exit signal is sent to <c><anno>Pid</anno></c>,
-          which unconditionally exits with exit reason <c>killed</c>.</p>
+        <p>The following behavior applies if
+        <c><anno>Reason</anno></c> is the term <c>normal</c> and
+        <c><anno>Pid</anno></c> is the identifier of a process
+        <c>P</c> which is not the same as the process that invoked
+        <c>erlang:exit(Pid, normal)</c> (the behavior when a process
+        sends a signal with the <c>normal</c> reason to itself is
+        described in the warning):</p>
+        <list type="bulleted">
+          <item>If <c>P</c> is <seemfa
+          marker="#process_flag/2">trapping exits</seemfa>, the exit signal is
+          transformed into a message <c>{'EXIT', From, normal}</c>,
+          where <c>From</c> is the process identifier of the process
+          that sent the exit signal, and delivered to <c>P</c>'s
+          message queue.</item>
+          <item>The signal has no effect if <c>P</c> is not trapping
+          exits.</item>
+        </list>
+        <p>If <c><anno>Reason</anno></c> is the atom <c>kill</c>, that
+        is, if <c>exit(<anno>Pid</anno>, kill)</c> is called, an
+        untrappable exit signal is sent to the process that
+        is identified by <c><anno>Pid</anno></c>, which
+        unconditionally exits with exit reason <c>killed</c>.  The
+        exit reason is changed from <c>kill</c> to <c>killed</c> so
+        that the <c>kill</c> signal will not cascade to all linked
+        processes.
```

Comment:
The "The exit reason is changed from `kill` to `killed` so that the `kill` signal will not cascade to all linked processes." sentence isn't really true.

There are exit signals with exit reason `kill` that does not unconditionally terminate the receiver. See comment under `exit/1`. If `kill` reason was kept `ProcessCompleted` would in this case be `true` on all exit signals propagated due to links. That is, the propagated exit signals would not unconditionally terminate receivers.

Reply:
I have updated the text 3b5bfbb. @rickard-green can you check my update?